src/components/tabs/LayersTab.vue

```diff
@@ -9,6 +9,21 @@
           official PyTorch documentation
         </a>.
       </p>
+      <ExpandablePanel
+        :name="overviewCategories.Model"
+        v-show="searchString === '' || renderedModelEditors.length > 0"
+      >
+        <div class="msg" v-show="modelEditors.length <= 1">No Models Created</div>
```

Comment:
I don't think "No Models Created" is the right text here, since it will be showed when you're creating a Model. Perhaps "No Models Available" or something?

Reply:
Will do